probe-rs/targets/nRF51 Series.yaml

```diff
@@ -107,6 +111,7 @@ variants:
           page_size: 1024
           erased_byte_value: 255
   - name: nRF51822_xxAB
+    part: 0x1
```

Comment:
Does the nRF51822_xxAB have the same part number as the xxAC variant?

Reply:
I am not sure. We should definitively handle this.